README.adoc

```diff
@@ -19,20 +19,20 @@
 
 Explore how to access a simple RESTful web service and consume its resources in Java.
 
-
 // =================================================================================================
 // Introduction
 // =================================================================================================
 
 == What you'll learn
 
-You will learn how to access a REST service and deserialize the returned JSON resources containing a
-list of artists and their albums using two different approaches. The first approach consists of using
-the JSON-P API to process the JSON. The second approach focuses on using a JSON binding library like
-Jackson to convert JSON messages into Java objects directly.
+You will learn how to access a REST service, serialize a java object containing a
+list of artists and their albums, and deserialize the returned JSON resources using
+two different approaches. The first approach consists of using JSON-B (binding) API
+to convert JSON messages into Java objects directly.  The second approach consists of
+using the JSON-P (JSON with Padding) API to process the JSON.
```

Comment:
- I'd say, "**the** JSON-B..."
- We try to avoid parenthesis. Could we either remove the content in parenthesis or somehow rewrite these sentences to include the content but remove the parenthesis?
- Maybe also move the adverb like so: "...convert JSON messages directly into Java objects."
- Remove the extra space between the sentences on line 31.